app/views/dashboards/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render partial: 'introduction',
+    locals: {
+      new_link: buildable?, export_link: buildable?, list_link: false,
+      push_staging_link: buildable?, push_prod_link: buildable?
+    }
+%>
```

Comment:
I see ApplicationHelper#buildable? being used here, but if it is always true, then maybe this can be cleaned up a bit. I'm cool leaving it for now though.